server/src/main/java/org/candlepin/async/tasks/ExportJob.java

```diff
@@ -99,24 +98,44 @@ public Object execute(final JobExecutionContext jdata) throws JobExecutionExcept
     }
 
     /**
-     * Schedules the generation of a consumer export. This job starts immediately.
+     * Creates a JobBuilder configured to execute the export job. Callers may further manipulate
+     * the JobBuilder as necessary before queuing it.
      *
-     * @param consumer the target consumer
-     * @param cdnLabel the cdn label
-     * @param webAppPrefix the web app prefix
-     * @param apiUrl the api url
-     * @return a JobDetail representing the job to be started.
+     * @param consumer
+     *  the consumer to export
+     *
+     * @param owner
+     *  the owner performing the export
+     *
+     * @param cdnLabel
+     *  the CDN label to store in the meta file
+     *
+     * @param webAppPrefix
+     *  the URL pointing to the manifest's originating web application
+     *
+     * @param apiUrl
+     *  the API URL pointing to the manifest's originating candlepin API
+     *
+     * @param extensionData
+     *  data to be passed to the {@link ExportExtensionAdapter} when creating a new export of the
+     *  target consumer.
+     *
+     * @return
+     *  a JobBuilder configured to execute the export job
      */
-    public static JobBuilder scheduleExport(
-        final Consumer consumer,
-        final Owner owner,
-        final String cdnLabel,
-        final String webAppPrefix,
-        final String apiUrl,
-        final Map<String, String> extensionData
-    ) {
+    public static JobBuilder createJobBuilder(Consumer consumer, Owner owner, String cdnLabel,
+        String webAppPrefix, String apiUrl, Map<String, String> extensionData) {
+
+        if (consumer == null) {
+            throw new IllegalArgumentException("consumer is null");
+        }
+
+        if (owner == null) {
+            throw new IllegalArgumentException("owner is null");
+        }
+
         return JobBuilder.forJob(JOB_KEY)
-            .setJobName("export-" + Util.generateUUID())
+            .setJobName("export_manifest")
```

Comment:
We already have two fields for that: correlated service ID and the job ID.

This old method was basically a holdover/hack from how we used to differentiate. Our log format for the line you're thinking of looks like this:
```<pattern>%d{ISO8601} [thread=%thread] [%X{requestType}=%X{requestUuid}, org=%X{org}, csid=%X{csid}] %-5p %c - %m%n</pattern>```

In Pinsetter, this was populated as follows:
```
MDC.put("requestType", "job");
MDC.put("requestUuid", context.getJobDetail().getKey().getName());
```
Note that we're setting the job name as the request UUID.

In JobManager, we're doing this instead:
```
MDC.put("requestType", "job");
MDC.put("requestUuid", status.getId());
```

If we were to keep the old-style log entries, I'd rather build the string on the fly using the name and ID, rather than forcing that to be done by the users creating the jobs. But, adjustment period aside, I think we'll be okay on this front.